avocado/core/tree.py

```diff
@@ -104,6 +104,10 @@ def __eq__(self, other):
                     return False
             return True
 
+    def __ne__(self, other):
+        """ Inverted eq """
+        return not self == other
```

Comment:
I don't think so, or at least it does not work properly in all cases. I added this commit because it didn't work later in the code...